keras_retinanet/bin/train.py

```diff
@@ -305,6 +307,13 @@ def main(args=None):
     # create the generators
     train_generator, validation_generator = create_generators(args)
 
+    if args.backbone == 'resnet':
+        from ..models.resnet import resnet_retinanet as retinanet, custom_objects, download_imagenet
```

Comment:
Where does custom_objects come from? The import was removed?

Reply:
Likewise for download_imagenet.

Reply:
Wrap these imports in try and excepts, maybe? As it could fail to import, and that should count as 'NotImplemented' too?